.github/workflows/build-ubuntu.yaml

```diff
@@ -105,11 +113,29 @@ jobs:
       - name: Install cached non packaged dependencies
         if: steps.build-cache-restore-step.outputs.cache-hit != 'true' # Variable type is string, thus using quotes
         working-directory: deps
-        run: ../docker/build_deps.sh
+        run: |
+          ${{ inputs.cpp03 }} ]] && ../docker/build_deps.sh --cpp03 || ../docker/build_deps.sh
 
       # Build BlazingMQ
+      - name: Build BlazingMQ [C++03]
+        if: inputs.cpp03 == true && steps.build-cache-restore-step.outputs.cache-hit != 'true' # Variable type is string, thus using quotes
+        env:
+          PKG_CONFIG_PATH: /usr/lib/x86_64-linux-gnu/pkgconfig:/opt/bb/lib64/pkgconfig
+        run: |
+          cmake -S . -B build/blazingmq -G Ninja \
+            -DCMAKE_TOOLCHAIN_FILE=${{ github.workspace }}/deps/srcs/bde-tools/BdeBuildSystem/toolchains/linux/gcc-default.cmake \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DBDE_BUILD_TARGET_SAFE=ON \
+            -DBDE_BUILD_TARGET_64=ON \
+            -DBDE_BUILD_TARGET_CPP03=ON \
```

Comment:
I think the same can be achieved by passing `ufid`:
`./configure [...] --ufid opt_64_cpp17` (need to check what `opt` means)

Having a property `cxx_standard` and `ufid` should allow to merge both workflow cases into one